.github/workflows/restore-database.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Perform Django dbrestore and mediarestore
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: string
+        # dev, staging, production
+      dbbackup-name:
+        required: true
+        type: string
+      mediabackup-name:
+        required: true
+        type: string
+
+jobs:
+  restore-database:
+    name: Restore ${{ inputs.environment }} database
```

Comment:
We shouldn't have to target anything.

Meta shares the `backups` bucket to staging, so `prod and staging` have access to the bucket when you `bind-service`
Dev has its own bucket, which, we can share to `preview` so then `preview and dev` will have access to the bucket when you `bind-service`

Reply:
How do we specify that we are taking the backup from prod and applying to staging? Don't we need to say that the restore should actuallt run in the staging space?
One more thing. The dbbackup and mediabackup taken at a point in tims need to be restored together.

Reply:
By the file name. `production-db-backup-2023-09-25-1714`

If you are in the `staging` space, and the backups bucket is shared to `staging` space, then you bind the bucket to the app, when you run `python manage.py dbrestore -i production-db-backup-2023-09-25-1714.psql.bin` it will look in the s3 for that file, and then apply it in that space.

The media and db are restored together in `restore_database.sh`